neuralmonkey/vocabulary.py

```diff
@@ -43,39 +43,48 @@ def _is_special_token(word: str) -> bool:
             or word == UNK_TOKEN)
 
 
-def from_wordlist(path: str, encoding: str = "utf-8") -> 'Vocabulary':
+def from_file(path: str) -> 'Vocabulary':
```

Comment:
tady nemá bejt `path` ale `*args` a `**kwargs`, aby bylo jedno, s jakými argumenty se to volá.